opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexDatabase.java

```diff
@@ -1371,7 +1370,13 @@ public void removeIndexChangedListener(IndexChangedListener listener) {
                 iter = terms.iterator(); // init uid iterator
             }
             while (iter != null && iter.term() != null) {
-                files.add(Util.uid2url(iter.term().utf8ToString()));
+                String value = iter.term().utf8ToString();
+                if (value.isEmpty()) {
+                    iter.next();
+                    continue;
+                }
+
+                files.add(Util.uid2url(value));
```

Comment:
why is this necessary? it wasn't used before?

Reply:
The first term is always empty, it never seem to have worked correctly.

Reply:
Aha, so I guess nobody tried this code for a long time.